UnitTests/MParticleTestsSwift.swift

```diff
@@ -171,4 +172,30 @@ class MParticleTestsSwift: XCTestCase {
             """
         )
     }
+       
+    func testLogEventCallbackDataFilterNotSet() {
+        XCTAssertNil(mparticle.dataPlanFilter)
+        mparticle.logEventCallback(MPEvent(), execStatus: .success)
+        
+        XCTAssertNil(receivedMessage)
+    }
+    
+    func testLogEventCallbackrDataFilterSetDataFilterReturnNil() {
```

Comment:
Function naming typo here
```suggestion
testLogEventCallbackDataFilterSetDataFilterReturnNil() {